### ISSUE TITLE
How to prepare data for pretrian for TTS?

### ISSUE THREAD

**Issue by fmsky** [canopyai/Orpheus-TTS#156]
Hello, 
       As you metioned in pretrain/readme.md, "If your model is just trained for TTS start with 1:1 and gradually decrease to 0:1." What does this mean? What are effective strategies for designing loss functions in TTS systems?

**Comment by karayakar** [comment]
> Hello, As you metioned in pretrain/readme.md, "If your model is just trained for TTS start with 1:1 and gradually decrease to 0:1." What does this mean? What are effective strategies for designing loss functions in TTS systems?

You can use this tool

**Comment by Sundragon1993** [comment]
@karayakar, do you have the code to prepare the QA dataset? Not the Parquet one

**Comment by karayakar** [comment]
> [@karayakar](https://github.com/karayakar), do you have the code to prepare the QA dataset? Not the Parquet one

It is already in the repo 
After creating parquet file you can use 
https://github.com/karayakar/OrpheusTTS-ParquetDatasetCreator/blob/master/pythonScripts/createdataset.py

**Comment by Sundragon1993** [comment]
Thank you @karayakar but the given code is for TTS dataset preparation, not the QA dataset which is essential for the pretraining phase.

**Comment by karayakar** [comment]
> Thank you [@karayakar](https://github.com/karayakar) but the given code is for TTS dataset preparation, not the QA dataset which is essential for the pretraining phase.

You can modify the script for text field but not necessary... Which I did pretrain as is..

https://huggingface.co/spaces/Karayakar/Orpheus-TTS-Turkish

**Comment by Sundragon1993** [comment]
@karayakar may I ask if the cloning feature work on your pretrained model? Did you pass the same TTS and QA dataset in the config file?

**Comment by karayakar** [comment]
> [karayakar](https://github.com/karayakar) may I ask if the cloning feature work on your pretrained model? Did you pass the same TTS and QA dataset in the config file?

cloning doesn't work.

**Comment by ukemamaster** [comment]
@fmsky Did you manage to pre-train only for TTS? If yes, could you please share your experience?

**Comment by ukemamaster** [comment]
> > Thank you [@karayakar](https://github.com/karayakar) but the given code is for TTS dataset preparation, not the QA dataset which is essential for the pretraining phase.
> 
> You can modify the script for text field but not necessary... Which I did pretrain as is..
> 
> https://huggingface.co/spaces/Karayakar/Orpheus-TTS-Turkish
> 
> 

Hi @karayakar can you please explain how to modify your script to generate QA dataset? The input metadata file for TTS dataset is audio_filename and text pairs. In case of QA, how it should look like?

**Comment by zzhdbw** [comment]
https://github.com/canopyai/Orpheus-TTS/issues/265#issuecomment-3222729749
